apis/apps/v1alpha1/nodeimage_types.go

```diff
@@ -37,7 +37,9 @@ type ImageSpec struct {
 	PullSecrets []ReferenceObject `json:"pullSecrets,omitempty"`
 
 	// Tags is a list of versions of this image
-	Tags []ImageTagSpec `json:"tags"`
+	// +patchMergeKey=tag
+	// +patchStrategy=merge
+	Tags []ImageTagSpec `json:"tags"  patchStrategy:"merge" patchMergeKey:"tag"`
```

Comment:
This type is not needed because it is not issued externally.

Reply:
done